website/docs/guides/enhanced-region-support.html.markdown

```diff
@@ -0,0 +1,504 @@
+---
+subcategory: ""
+layout: "aws"
+page_title: "Terraform AWS Provider Enhanced Region Support"
+description: |-
+  Enhanced Region support with the Terraform AWS Provider.
+---
+
+# Enhanced Region Support
+
+Most AWS resources are Regional – they are created and exist in a single AWS Region, and to manage these resources the Terraform AWS Provider directs API calls to endpoints in the Region. The AWS Region used to provision a resource with the provider is defined in the [provider configuration](https://developer.hashicorp.com/terraform/language/providers/configuration) used by the resource, either implicitly via [environment variables](https://registry.terraform.io/providers/hashicorp/aws/latest/docs#environment-variables) or [shared configuration files](https://registry.terraform.io/providers/hashicorp/aws/latest/docs#shared-configuration-and-credentials-files), or explicitly via the [`region` argument](https://registry.terraform.io/providers/hashicorp/aws/latest/docs#region).
```

Comment:
Nit: Obviously we are going to say "region" quite a lot in this guide and its probably worth a bit of discussion as to how we use it consistently. In this doc we have:

- Region, Regional, Regions (used when discussing the concept)
- AWS Region (used when introducing the concept
- `region` (used when referring to the specific argument)

Generally I think we are doing it right, my only question is whether Region and friends should be capitalized?

Reply:
We'll only use _AWS Region_ in the first sentence.
In this guide we are using "uppercase R" Region as that's what AWS do (see https://aws.amazon.com/about-aws/global-infrastructure/). Outside this guide I'm pretty sure we are using "lowercase R" region in many places.

Reply:
Hopefully we are at least consistent in this guide.